AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -169,6 +173,32 @@ - (void)signIn {
   }
 }
 
+- (void)signOut {
+  @synchronized(self) {
+    if (![self canBeUsed]) {
+      return;
+    }
+    if ([MSAuthTokenContext sharedInstance].authToken != nil) {
+      MSLogInfo([MSIdentity logTag], @"User sign-out succeeded.");
+      [[MSAuthTokenContext sharedInstance] clearAuthToken];
+      if (self.clientApplication != nil) {
+        MSALAccount *account = [self retrieveAccountWithAccountId:[self retrieveAccountId]];
+        if (account != nil) {
+          NSError *error;
+          [self.clientApplication removeAccount:account error:&error];
+          if (error) {
+            MSLogError([MSIdentity logTag], @"Couldn't remove account: %@", error.localizedDescription);
```

Comment:
It's a little confusing when we fail to delete account info from storage during sign-out that we log a success message (line 182) and an error message (line 190).

I think it make more sense to log a warning here. It's not a big deal if we fail to delete here because it will get overwritten on the next sign-in.